nilearn/plotting/find_cuts.py

```diff
@@ -269,3 +273,59 @@ def find_cut_slices(img, direction='z', n_cuts=12, spacing='auto'):
     cut_coords.sort()
 
     return _transform_cut_coords(cut_coords, direction, affine)
+
+
+def find_parcellation_atlas_cut_coords(label_img):
+
+    """ Grab coordinates of center of mass of 3D parcellation atlas
+
+    Parameters
+    ----------
+    label_img: 3D Nifti1Image
+        A brain parcellation atlas
+
+    Returns
+    -------
+    labels_list: List
+        Label region values
+    coord_list: List
+        Center coordinates for label regions
+    """
+
+    # grab data and affine
+    label_img = check_niimg(label_img)
+    label_data = label_img.get_data()
+    label_affine = label_img.get_affine()
+
+    # grab number of unique values in 3d image
+    label_unique = np.unique(label_data)[1:]
```

Comment:
This should be replaced by relying on the parameter that specifies the background label.